src/Ubidots.cpp

```diff
@@ -38,6 +37,7 @@ Ubidots::Ubidots(UbiToken token, UbiApn apn, UbiApn apnUser, UbiApn apnPass, Ubi
 void Ubidots::_builder(UbiToken token, UbiApn apn, UbiApn apnUser, UbiApn apnPass, UbiServer server = UBI_INDUSTRIAL,
                        IotProtocol iotProtocol = UBI_TCP) {
   _iotProtocol = iotProtocol;
+  sprintf(_defaultDeviceLabel, DEFAULT_DEVICE_LABEL);
```

Comment:
the variable `_defaultDeviceLabel` is not necessary, just modify the send() method as follows:

```
bool Ubidots::send() { return send(DEFAULT_DEVICE_LABEL, DEFAULT_DEVICE_LABEL); }
```